x-pack/plugins/observability_solution/infra/public/components/asset_details/template/flyout.tsx

```diff
@@ -31,7 +31,7 @@ export const Flyout = ({
   const {
     services: { telemetry },
   } = useKibanaContextForPlugin();
-  const { metadata, loading: metadataLoading } = useMetadataStateContext();
+  const { metadata } = useMetadataStateContext();
```

Comment:
After removing the prop `hasSystemIntegration` as mentioned in the previous comment we can also remove this one